packages/platform/platform-fastify/test/test-session.integration.spec.ts

```
@@ -0,0 +1,118 @@
+    return Promise.resolve(user);
+  }
+
+  @Get("/logout")
+  @Returns(204)
+  logout(@Req() req: any) {
+    switch (this.platformName) {
+      case "koa":
+        return Promise.resolve((req.ctx.session = null));
+      case "fastify":
+      case "express":
+        delete req.session.user;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)